### PR TITLE
[css-flexbox] New test for iframes as flex items

### PR DESCRIPTION
**Pull request by svillar**
This comes from [this WebKit bug](https://bugs.webkit.org/show_bug.cgi?id=234361). A recent commit in WebKit regressed those cases where non-image replaced elements were sized as it they had an intrinsic ratio when they really don't.

One of those examples of replaced elements that are not images is `<iframe>`s. They have a default 300x150px size but that does not imply that they have an intrinsic ratio of 2.

### CODE REVIEW

File: css/css-flexbox/flexbox-iframe-intrinsic-size-ref.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<style>
+div {
+    height: 400px;
+    border: 0;
```
Comment (mrego) [inline css/css-flexbox/flexbox-iframe-intrinsic-size-ref.html:5]: You don't need this border.

File: css/css-flexbox/flexbox-iframe-intrinsic-size.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com"/>
+<link rel="help" href="https://drafts.csswg.org/css-flexbox/#algo-main-item" title="Part E">
+<link rel="help" href="https://www.w3.org/TR/css-sizing-3/#intrinsic-sizes" title="For boxes with no preferred aspect ratio">
```
Comment (mrego) [inline css/css-flexbox/flexbox-iframe-intrinsic-size.html:4]: Could you add a "meta assert" explaining the purpose of the test?

File: css/css-flexbox/flexbox-iframe-intrinsic-size.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (mrego) [inline css/css-flexbox/flexbox-iframe-intrinsic-size.html:1]: Nit: Add -001 suffix.

**Review by mrego — APPROVED** [review]
LGTM, with some comments.

File: css/css-flexbox/flexbox-iframe-intrinsic-size.html (hunk 2 above)
Comment (svillar) [reply css/css-flexbox/flexbox-iframe-intrinsic-size.html:4]: I thought it was enough with the <p> explaining the result but yeah I can add it.

File: css/css-flexbox/flexbox-iframe-intrinsic-size.html (hunk 3 above)
Comment (svillar) [reply css/css-flexbox/flexbox-iframe-intrinsic-size.html:1]: Sure.

**Pull request merged**